test/server/watchFiles-option.test.js

```diff
@@ -298,6 +302,21 @@ describe("'watchFiles' option", () => {
       fs.truncateSync(file);
     });
 
+    it('should pass correct options to chokidar config', () => {
+      expect(chokidarMock).toHaveBeenCalledWith(file, {
+        ignoreInitial: true,
+        persistent: true,
+        followSymlinks: false,
+        atomic: false,
+        alwaysStat: true,
+        ignorePermissionErrors: true,
+        // eslint-disable-next-line no-undefined
+        ignored: undefined,
+        usePolling: true,
+        interval: 400,
+      });
+    });
```

Comment:
Let's add do this check in each test, to avoid future regressions, also let's add test:
- `poll: true`
- `usePolling: true` without interval
- `usePolling: false` and `poll: true` in this case we should prefer use `false` value (maybe need typeof)
- `usePolling: true` and `interval: 200` and `poll: 400` (should be preferred `200`)

Reply:
yep wip

Reply:
done all